src/Lib.hs

```diff
@@ -356,6 +358,37 @@ inspectAttachments ticketInfo attachments = runMaybeT $ do
 
     pure $ inspectAttachment config ticketInfo att
 
+-- | Inspection of the local zip.
+-- This function prints out the analysis result on the console.
+inspectLocalZipAttachment :: FilePath -> App ()
+inspectLocalZipAttachment filePath = do
+
+    config          <- ask
+    printText       <- asksIOLayer iolPrintText
+
+    -- Read the zip file
+    fileContent     <- liftIO $ BS.readFile filePath
```

Comment:
I think we should use `readFile` from ioLayer. If we try to implement test case on this function, it'lll try to read an external file which probably does not exist. Using ioLayer can prevent this.

Reply:
Yes, I know that - that's why I added these functions in the first place. However, this function is an `LByteString` while the "regular" one is a `ByteString`.
Since we don't want to introduce another function for lazy reading and the rest of the functions are pretty decently tested, this should be good.

Reply:
Got it, thank you!